llama-stack-client-kotlin-core/src/main/kotlin/com/llama_stack_client/api/models/AgentCreateParams.kt

```diff
@@ -3874,7 +3874,7 @@ constructor(
                             private var additionalProperties: MutableMap<String, JsonValue> =
                                 mutableMapOf()
 
-                            internal fun from(type: Type) = apply {
+                            internal fun from(type: QueryGeneratorConfig.Type) = apply {
```

Comment:
I'm wondering if these patches can be auto-generated? Otherwise, we may need to reapply the same fix everytime we generate & sync the SDK w/ Stainless.

Reply:
I haven't explored the autogenerated patch, but prob something we can look into more. For now those fixes are generated by Android Studio, by following the recommended fixes.